src/domain/occurrence/eventOccurrenceForm/ValidationSchema.ts

```diff
@@ -95,3 +98,41 @@ export default Yup.object().shape({
     ),
   oneGroupFills: Yup.boolean(),
 });
+
+const testEnrolmentEndDays = (
+  enrolmentStart: Date,
+  enrolmentEndDays: number,
+  schema: Yup.DateSchema<Date>
+) => {
+  const minDate =
+    enrolmentEndDays > 0
+      ? addDays(enrolmentStart, enrolmentEndDays)
+      : enrolmentStart;
+  minDate.setHours(0, 0, 0, 0);
+  return schema.min(minDate, () => ({
+    key: VALIDATION_MESSAGE_KEYS.DATE_MIN,
+    min: formatDate(minDate),
+  }));
+};
+
+export const testDateWithEnrolmentValidationSchema = (
```

Comment:
I feel like this could be just named `getValidationSchema`. Maybe there could be comment in the function to clarify the validation logic

Reply:
changed and commented.